resources/zib-Patient.xml

```diff
@@ -0,0 +1,607 @@
+      <alias value="Naamgegevens" />
+      <type>
+        <code value="HumanName" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/zib-NameInformation" />
+      </type>
+      <mapping>
+        <identity value="zib-contactperson-v3.4-2020EN" />
+        <map value="NL-CM:3.1.4" />
+        <comment value="NameInformation" />
+      </mapping>
+    </element>
+    <element id="Patient.contact.telecom">
+      <path value="Patient.contact.telecom" />
+      <slicing>
+        <discriminator>
+          <type value="profile" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+      <short value="ContactInformation" />
+      <definition value="The contact’s telephone number and/or e-mail address." />
+      <comment value="The cardinality mismatch between the zib (0..1) and FHIR (0..*) is explained by the containers TelephoneNumbers (0..*) and EmailAddresses (0..*) inside the ContactInformation model. The FHIR datatype ContactPoint does not have these containers." />
+      <alias value="Contactgegevens" />
+      <mapping>
+        <identity value="zib-contactperson-v3.4-2020EN" />
+        <map value="NL-CM:3.1.6" />
+        <comment value="ContactInformation" />
+      </mapping>
+      <mapping>
+        <identity value="zib-contactinformation-v1.2-2020EN" />
+        <map value="NL-CM:20.6.1" />
+        <comment value="ContactInformation" />
+      </mapping>
+    </element>
+    <element id="Patient.contact.telecom:telephoneNumbers">
+      <path value="Patient.contact.telecom" />
+      <sliceName value="telephoneNumbers" />
+      <type>
+        <code value="ContactPoint" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/zib-ContactInformation-TelephoneNumbers" />
+      </type>
+    </element>
+    <element id="Patient.contact.telecom:e-mailAddresses">
+      <path value="Patient.contact.telecom" />
+      <sliceName value="e-mailAddresses" />
+      <type>
+        <code value="ContactPoint" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/zib-ContactInformation-E-mailAddresses" />
+      </type>
+    </element>
+    <element id="Patient.contact.address">
+      <path value="Patient.contact.address" />
+      <short value="AddressInformation" />
+      <definition value="Contact’s address information." />
+      <alias value="Adresgegevens" />
+      <type>
+        <code value="Address" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/zib-AddressInformation" />
+      </type>
+      <mapping>
+        <identity value="zib-contactperson-v3.4-2020EN" />
+        <map value="NL-CM:3.1.5" />
+        <comment value="AddressInformation" />
+      </mapping>
+    </element>
+    <element id="Patient.communication">

```

Comment:
Deze mapping is lastig, kan je hier drie slices van maken voor de drie zib-concepten?

Reply:
Nice!